client/src/components/CRM/Contacts/ContactOneResult.tsx

```diff
@@ -1,272 +1,271 @@
-            <div className='font-semibold'>
-               Customer Address:
-            </div>
-            <div data-testid = 'customer-address'>{address}</div>
-          </div>
-          <div className='flex flex-row gap-3 text-lg mt-2 w-full'>
-            <div className='font-semibold'>
-               Newsletter:
-            </div>
-            <div data-testid = 'customer-newsletter'>{'' + newsletter}</div>
-          </div>
-          <div className='flex flex-row gap-3 text-lg mt-2 w-full'>
-            <div className='font-semibold'>
-               Donorbadge:
-            </div>
-            <div data-testid = 'customer-donorbadge'>{'' + donorbadge}</div>
-          </div>
-          <div className='flex flex-row gap-3 text-lg mt-2 w-full'>
-            <div className='font-semibold'>
-               Seating Accommodation:
-            </div>
-            <div data-testid = 'customer-accommodation'>{seatingaccom}</div>
-          </div>
-          <div className='flex flex-row gap-3 text-lg mt-2 w-full'>
-            <div className='font-semibold'>
-               VIP:
-            </div>
-            <div data-testid = 'customer-vip'>{'' + vip}</div>
-          </div>
-          <div className='flex flex-row gap-3 text-lg mt-2 w-full'>
-            <div className='font-semibold'>
-               Volunteer List:
-            </div>
-            <div data-testid = 'customer-volunteer'>{'' + volunteerlist}</div>
-          </div>
+export const ContactOrder = ({order}: {order: any}): ReactElement => {
+  const {orderdate, orderid, ordertime, ordertotal, refund_intent, orderitems} =
+    order;
+
+  const date = new Date(
+    `${toDateStringFormat(orderdate)}T${ordertime.split('T')[1].slice(0, 8)}`,
+  );
 
-        </div>
+  return (
+    <section className='w-full bg-white shadow-lg border border-zinc-300 rounded-lg mb-4 p-5 text-zinc-600'>
+      <header className='flex gap-2 items-start justify-between mb-4'>
+        <h2 className='text-2xl font-semibold'>Order {orderid}</h2>
+        <svg
+          xmlns='http://www.w3.org/2000/svg'
+          className='h-7 w-7'
+          fill='none'
+          viewBox='0 0 24 24'
+          stroke='currentColor'
+          strokeWidth={2}
+        >
+          <path
+            strokeLinecap='round'
+            strokeLinejoin='round'
+            d='M15 5v2m0 4v2m0 4v2M5 5a2 2 0 00-2 2v3a2 2 0 110 4v3a2 2 0 002 2h14a2 2 0 002-2v-3a2 2 0 110-4V7a2 2 0 00-2-2H5z'
+          />
+        </svg>
+      </header>
+      <div className='grid md:grid-cols-2'>
+        <article>
+          <p className='flex flex-row gap-3 text-lg w-full'>
+            <span className='font-semibold'>Order Date:</span>
+            <span>{format(date, 'MMM dd, yyyy')}</span>
+          </p>
+          <p className='flex flex-row gap-3 text-lg mt-1 w-full'>
+            <span className='font-semibold'>Order Time:</span>
+            <span>{format(date, 'h:mm a')}</span>
```

Comment:
We should include the timezone here and with the `eventDate` in the order item

Reply:
It looks like a timezone isn't stored in the database with the time stamp (it is just converted to UTC). And when the time is displayed to the user on the frontend, it is converted to their local timezone (so PST in most cases). Do we just want to display the user's local timezone?

Reply:
Hm it does look like for either of these timestamps they don't get saved with the timezone.. it also doesn't look like they are not converted to local time. If I create an order at 2pm my time (PST) then it gets saved as `14:00:00+00:00` and displayed back as 2pm. Maybe I'm missing something about these timestamps, but I would expect it to get saved as `22:00:00-08:00` where `22:00:00` is the UTC time and `-08:00` is the timezone offset. Either way, I'm ok with just keeping an eye on it and worrying about it if it starts causing problems

Reply:
I got those backwards my bad, I would expect the timestamp to be `14:00:00-08:00` where `14:00:00` is the local time PST and `-08:00` is the offset from UTC time

Reply:
Yeah, that's what we'd like, but it looks like were saving the time as a DateTime where it takes the form `1970-01-01T04:05:06` and then we're just extracting the time.